configs/dev-emulator-program-tripconfirm.nrel-op.json

```diff
@@ -0,0 +1,67 @@
+{
+    "version": 1,
+    "ts": 1655143472,
+    "intro": {
+        "program_or_study": "program",
+        "start_month": "9",
+        "start_year": "2020",
+        "mode_studied": "e-bike",
+        "program_admin_contact": "K. Shankari (k.shankari@nrel.gov)",
+        "deployment_partner_name": "National Renewable Energy Laboratory (NREL)",
+        "translated_text": {
+            "en": {
+                "deployment_name": "Development environment (program)",
+                "summary_line_1": "allows developers to work on the OpenPATH app",
+                "summary_line_2": "connect to a local server",
+                "summary_line_3": "and ensure that end-to-end program features work.",
+                "short_textual_description": "OpenPATH is the only open source platform for instrumenting mobility patterns. We welcome contributions and pull requests.",
+                "why_we_collect": "So that you can see how OpenPATH works",
+                "research_questions": ["Comparing the open-access group (no intervention) and programs that provide interventions.", "General monitoring of mobility trends for travel behavior, including shifts to travel behavior and the introduction of new mobility modes."]
+            }
+        }
+    },
+    "survey_info": {
+      "buttons": {
+        "trip-notes": {
+          "surveyName": "TimeUseSurvey",
+          "not-filled-in-label": {
+            "en": "Add Activity",
+            "es": "Añadir actividad"
+          },
+          "filled-in-label": {
+            "en": "",
+            "es": ""
+          }
+        },
+        "place-notes": {
+          "surveyName": "TimeUseSurvey",
+          "not-filled-in-label": {
+            "en": "Add Activity",
+            "es": "Añadir actividad"
+          },
+          "filled-in-label": {
+            "en": "",
+            "es": ""
+          }
+        },
+        "trip-labels": {
+          "survey-name": "TripConfirmSurvey",
+          "not-filled-in-label": {
+            "en": "Add trip details",
+            "es": "Agregar detalles del viaje"
+          },
+          "filled-in-label": {
+            "en": "Submitted",
+            "es": "Enviado"
+          }
+        }
```

Comment:
This should not be here, at least at this point. The trip confirm survey and the user profile survey are "special".
- trip confirm: there is only one per trip
- user profile: there is only one per user

This is also a **regression** wrt the TripConfirmSurvey because it is not using the more sophisticated `filled-in-label` from `LABEL_FUNCTIONS`.

That's why we decided to have different types of surveys in the first place.

Reply:
As I indicated in [that really long meeting today morning](https://github.com/e-mission/e-mission-phone/pull/915#issuecomment-1366063519), there are two separate versions of the trip label survey (`MULTILABEL` and `ENKETO`). The dynamic config should simply state which one we use. If we choose ENKETO, it will always be the `TripConfirmSurvey` defined in `surveys.json`